README.md

```diff
@@ -0,0 +1,66 @@
+# CDN Integration with Amzon Cloudfront
+
+
+Adds support for caching asset and media urls thru a cdn instance on amazon cloudfront.
+
+## Configuration
+
+### For single domain
+	
+	;bind kwf events
+	eventSubscribers.awscdn = KwfAwsCdn_Events
+	;bind clear cache
+	clearCacheTypes.assetsAwsCdn = KwfAwsCdn_ClearCacheTypeAssets
+	; individual cdn config
+	awscdn.access_key_id = [YOUR-AWS-KEY]
+	awscdn.secret_access_key_id = [YOUR-AWS-SECRET_KEY]
+	awscdn.distribution_id = [YOUR-CDN-DISTRIBUTION-ID]
+	awscdn.domain = [YOUR-CDN-WEB-DOMAIN]
+
+
+### For multi domain
+
+	;bind kwf events
+	eventSubscribers.awscdn = KwfAwsCdn_Events
+	;bind clear cache
+	clearCacheTypes.assetsAwsCdn = KwfAwsCdn_ClearCacheTypeAssets
+	; individual cdn config
+	awscdn.access_key_id = [YOUR-AWS-KEY]
+	awscdn.secret_access_key_id = [YOUR-AWS-SECRET_KEY]
+	kwc.domains.my.awscdn.distribution_id = [YOUR-CDN-DISTRIBUTION-ID]
+	kwc.domains.my.awscdn.domain = [YOUR-CDN-WEB-DOMAIN]
+
+
+
+[more info on credentials](https://docs.aws.amazon.com/aws-sdk-php/v2/guide/credentials.html#passing-credentials-into-a-client-factory-method)
+
+## Compatibility
+
+Does not work in combination with [https://github.com/koala-framework/kwf-varnish
+](). Kwf-Varnish has the same purpose as this package but uses a different strategy.
+Be sure to turn varnish off:
+
+### For single domain
+
+	varnish.domain = false
+
+### For multi domain
+
+	kwc.domains.hu.varnish.domain = false
+
+
+## Test enviromnent
+
+Kwf Prelogin must be disabled. Otherwise Amazon Cloudfront cannot properly cache the site.
```

Comment:
Wie ist das am test server?

Reply:
Man bekommt einen prompt um den Prelogin einzugeben wenn ein Image vom cdn eingebunden ist.
Würde man im Prompt die Credentials eingeben hilft das auch nicht, weil ja kein Backend für die Authentification am cdn liegt. (Da liegt ja nur das HTML)

Reply:
Dann kann mans aber nicht testen. Die test cdn instanz ist dann umsonst, die Klont ja die testserver inhalte mit den auth-headern. Jeder request auf das cdn ist somit ein 403, wenn am test prelogin aktiv ist. Ich hab in Absprache mit franky den prelogin für my deaktiviert. Man kann das cdn auf test natürlich deaktivieren, dann haben wir aber eine black box auf prod.